index.css

```diff
@@ -82,3 +81,149 @@ body {
 .social-media-spacer {
   margin-left: 20px;
 }
+
+.grid-item {
+  display: block;
+  box-shadow: 2px;
+  padding: 5px;
+  width: 300px;
+  border: 2px solid#DFE1E6;
+  border-radius: 10px;
+}
+
+.image-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.image-wrapper > img {
+  width: 250px;
+  border-radius: 30px;
+  align-content: center;
+}
+
+.grid-container {
+  margin-top: 150px;
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-left: 15px;
+  row-gap: 70px;
+}
+
+.grid-item > h1 {
+  font-weight: bold;
+  font-size: x-large;
+  color: #172b4d;
+}
+
+.work-details {
+  display: flex;
+  color: #7a869a;
+  align-items: center;
+  font-size: medium;
+  font-weight: bolder;
+}
+
+.first-detail {
+  color: #344563;
+  font-weight: bold;
+}
+
+.spacer {
+  width: 10px;
+  height: 10px;
+  border-radius: 100%;
+  background: #7a869a;
+}
+
+.margin-spacer {
+  margin-left: 10px;
+}
```

Comment:
Please use the button styling for your project for a good user interface and great interaction for visitors 😄 

This is the button styling you are to use :
![image](https://user-images.githubusercontent.com/20537712/235315145-d896b5c5-e456-4942-829b-a7981e773ad2.png)